test/tracer_integration/phoenix/endpoint_test.exs

```diff
@@ -232,7 +232,7 @@ defmodule TracerWithPhoenixEndpointTest do
 
     test "handles non-ASCII characters in path params" do
       assert capture_log(fn ->
-               call(Endpoint, :get, "/hello/%f0%9f%a4%af")
+               call(Endpoint, :get, "/hello/+%f0%9f%a4%af")
```

Comment:
@GregMefford was really lazy in how I tested this. the grownup way would probably be to extract the metadata stuff into their own module/little functions for unit testing, but I didn't want to come in here and rewrite a bunch of stuff =)